about-examples.md

```diff
@@ -92,4 +92,6 @@ module = "hello.wasm"
 route = "/"
 ```
 
-Then we start `wagi` with `wagi -c modules.toml`.
\ No newline at end of file
+Then we start `wagi` with `wagi -c modules.toml`.
```

Comment:
TODO (me): update this to use Spin.

Reply:
@radu-matei there are both Spin and Wagi instructions here. The Wagi part is just until we finish moving all of our examples over to Spin